server/src/main/java/com/objectcomputing/checkins/services/opportunities/OpportunitiesCreateDTO.java

```diff
@@ -0,0 +1,92 @@
+package com.objectcomputing.checkins.services.opportunities;
+
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+import java.time.LocalDate;
+
+@Introspected
+public class OpportunitiesCreateDTO {
+
+    @NotNull
+    @Schema(required = true, description = "name of the opportunity")
+    private String name;
+
+    @NotNull
+    @Schema(required = true, description = "description of the opportunity")
+    private String description;
+
+    @NotNull
+    @Schema(required = true, description = "link to the url")
+    private String url;
+
+    @NotNull
+    @Schema(required = true, description = "date for expiresOn")
+    private LocalDate expiresOn;
+
+
+    @NotNull
+    @Schema(required = true, description = "date for submittedOn")
+    private LocalDate submittedOn;
+
+    @NotNull
+    @Schema(required = true, description = "id of the associated member")
+    private UUID submittedBy;
```

Comment:
I should have noticed this before, but I don't think we want to send these in. This is whoever the current user is...and the current date.